add html for header and create svg sprite

olita16 opened the pull request.

olita16 committed:
add html for header and create svg sprite

olita16 committed:
add svg, js, html for footer and mobile menu

Commit:
Merge branch 'main' into headerfooter

sgoithw reviewed (changes requested):
По структурі html розмітки не бачу зауважень ) Код супер, все відповідно до ТЗ. Є пропозиція зробити Privacy Policy та Terms of Service лінками. Можна писати стилі і робити адаптацію під пристрої :)

olita16 committed:
Merge branch 'main' into headerfooter

olita16 committed:
add header/footer/mobile menu css js and roots

add styles CSS (header, footer and mobile menu)

add roots in variables
add folder javascript with 2 files(mobile-menu.js and mobile-open-menu.js)  and import in  main.js

olita16 committed:
Create .hintrc

Commit:
Merge branch 'headerfooter' of https://github.com/sgoithw/cheshire-cats into headerfooter

Commit:
Pretier runned

Commit:
Added some fixes

Commit:
Added some fixes
